charts/kube-state-metrics/Chart.yaml

```diff
@@ -2,18 +2,18 @@ apiVersion: v2
 name: kube-state-metrics
 description: Install kube-state-metrics to generate and expose cluster-level metrics
 keywords:
-- metric
-- monitoring
-- prometheus
-- kubernetes
+  - metric
```

Comment:
could we leave those formatting changes out? Unless there's a general decision to format all charts in a specific way, let's leave it as it is.

Reply:
Could I suggest adding something like an EditorConfig spec into the repo to keep formatting consistent?

Reply:
That sounds like a great idea, maybe also some sort of yamllint to enforce it during CI. :) 
(I'm not opposed to the change, just would want to have the same style enforced everywhere)

Reply:
I'll look at undoing the change here to get this merged and can look at opening another PR for the formatting if you'd like?

Reply:
Yeah, ideally start with an issue that describes what you'd like to set in terms of standards and how to enforce them on this repo.

Reply:
@mrueg I've reverted the _Chart.yaml_ formatting.